atmos.yaml

```diff
@@ -318,11 +318,20 @@ settings:
 
   # Terminal settings for displaying content
   terminal:
-    max_width: 120      # Maximum width for terminal output
-    pager: true         # Use pager for long output
-    timestamps: false   # Show timestamps in logs
-    colors: true       # Enable colored output
-    unicode: true      # Use unicode characters
+    max_width: 120        # Maximum width for terminal output
+    pager: true           # Use pager for long output
+    timestamps: false     # Show timestamps in logs
+    colors: true          # Enable colored output
+    unicode: true         # Use unicode characters
+
+    syntax_highlighting:
+      enabled: true
+      formatter: terminal   # Output formatter (e.g., terminal, html)
+      style: dracula        # Highlighting style
+      pager: true           # Enable pager
+      options:
```

Comment:
do we really need a separate section for `options` when there are only 2 valid options? I think we can move these up one level.